reana-ui/src/config.js

```diff
@@ -10,9 +10,8 @@
 
 const config = {
   name: "CERN REANA UI",
-  sign_up_email: "info@reana.io",
-  api: "http://reana-dev.cern.ch",
-  pooling_secs: 5,
+  api: "https://reana-dev.cern.ch",
```

Comment:
Hardcoding reana-dev ?

Reply:
I guess we will just leave `reana.cern.ch` for now? (UI only works at CERN for now)

In a more general aspect, this is a problem we need to solve for the future because this is something we can only change before build time, so if one sees REANA-UI on Docker-hub there are two identical tags one for DEV and another for QA which only differ on this field.

Reply:
Yeah, we can leave it easy for now, and the PROD seems better than DEV regarding outside effect.

BTW please also change the waffle bit, see `reana-commons` for example:

```diff
diff --git a/CONTRIBUTING.rst b/CONTRIBUTING.rst
index 2b45294..2dfa681 100644
--- a/CONTRIBUTING.rst
+++ b/CONTRIBUTING.rst
@@ -28,6 +28,6 @@ Each pull request should preserve or increase code coverage.
 Kanban
 ------
 
-We are using Kanban technique for keeping track of ongoing tasks. Please see our
-`Kanban board <https://waffle.io/reanahub/reana>`_ and look for issues that are
-labelled as "ready for work".
+We are using Kanban technique for keeping track of ongoing tasks. Please see
+our `Kanban boards <https://github.com/orgs/reanahub/projects>`_ and look for
+issues that are "ready for work".
```

We can use the same blob of text in every PR since Kanban boards are now organisation-wide.

Reply:
Done, thanks!